pkg/detectors/aws/aws_test.go

```diff
@@ -286,15 +269,44 @@ func TestAWS_FromChunk(t *testing.T) {
 					ExtraData: map[string]string{
 						"resource_type": "Access key",
 						"account":       "171436882533",
+						"is_canary":     "true",
+						"message":       "This is an AWS canary token generated at canarytokens.org, and was not set off; learn more here: https://trufflesecurity.com/canaries",
 					},
 				},
 			},
 			wantErr:               false,
 			wantVerificationError: true,
 		},
+		//// Needs a non-canary token to test this
```

Comment:
This test uses a fake HTTP client. It shouldn't matter what secret material gets used. Happy to chat about this synchronously to explain what it's doing and figure out a solution to whatever problem you were having.